src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Interop/LegacyExports.cs

```diff
@@ -0,0 +1,255 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    // this maps to src\mono\wasm\runtime\legacy\corebindings.ts
+    // the methods are protected from trimming by DynamicDependency on JSFunctionBinding.BindJSFunction
```

Comment:
Does it mean they will always be present?

Reply:
Next step is to make compile time condition to get rid of legacy interop. This class would not be included at all. That's why I separated it from `JavaScriptExports` class

Reply:
Not in this PR, I think